code/modules/spells/spell_types/conjure.dm

```diff
@@ -82,19 +82,35 @@
 	cooldown_min = 10
 	var/delete_old = TRUE //TRUE to delete the last summoned object if it's still there, FALSE for infinite item stream weeeee
 
-/obj/effect/proc_holder/spell/targeted/conjure_item/cast(list/targets, mob/user = usr)
```

Comment:
Am i srupid or did this get deleted

Reply:
idk wtf happened here but i'm pasting back the missing bits, think i was stupid and deleted it in botched attempt to make the summon spell